e2e/client/playwright/editor3.spec.ts

```diff
@@ -0,0 +1,28 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {restoreDatabaseSnapshot, s} from './utils';
+
+test('can add embeds', async ({page}) => {
+    await restoreDatabaseSnapshot();
+
+    const monitoring = new Monitoring(page);
+
+    await page.goto('/#/workspace/monitoring');
+
+    await monitoring.selectDesk('Sports');
+
+    await page.locator(
+        s('monitoring-group=Sports / Working Stage', 'article-item=test sports story'),
+    ).dblclick();
+
+    await expect(
+        page.locator(s('authoring', 'authoring-field=body_html')).getByRole('textbox'),
+    ).toHaveText('test sport story body');
+
+    await page.getByRole('button', {name: 'Embed'}).click();
+
+    await page.getByPlaceholder('Enter URL or code to embed').type('https://sourcefabric.org');
+
+    await page.locator(s('embed-controls', 'submit')).click();
+    await expect(page.getByText('https://sourcefabric.org')).toBeDefined();
```

Comment:
![image](https://github.com/superdesk/superdesk-client-core/assets/2076953/31951557-769b-43a1-a393-2227f0fb514d)

If we intercept the request and return a hardcoded response for testing we could make the test better by at least checking that title and description were fetched correctly.